### ISSUE TITLE
Question about training

### ISSUE THREAD

**Issue by wuqiyao20160118** [HelloRicky123/Siamese-RPN#34]
I use your training code with pretrained AlexNet to train SiamRPN, but the performance is pretty bad when only using VID dataset. I use the test code provided by [arbitularov](https://github.com/arbitularov/SiamRPN-PyTorch). When testing your pretrained model on OTB2015, the success score is 0.577 and the precision score is 0.772. However when using the saved model produced during my training, the performance is worser (success rate is about 0.45 and the precision score is about 0.61).  I'm wondering if the huge gap is origined from the lack of Youtube dataset or just the fluctuation of the training.

**Comment by HelloRicky123** [comment]
According to my experience, I can get about 0.53 AUC on OTB100 with only vid dataset and 0.63 AUC with vid and youtube-bb. So the gap is not small.

**Comment by wuqiyao20160118** [comment]
Thanks for your reply. I have tried to add GOT-10k dataset into training. It seems that the performance improves a little, but it does not perform as well as yours. (Both success score and percision score improves 0.05.) Maybe Youtube dataset is better? Or I should switch some training configuration like learning rate and some other hyperparameters?
